ipaserver/install/httpinstance.py

```diff
@@ -263,6 +265,20 @@ def enable_mod_nss_renegotiate(self):
         installutils.set_directive(paths.HTTPD_NSS_CONF, 'NSSRenegotiation', 'on', False)
         installutils.set_directive(paths.HTTPD_NSS_CONF, 'NSSRequireSafeNegotiation', 'on', False)
 
+    def enable_mod_nss_ocsp(self):
+        aug = augeas.Augeas()
+        ocsp_path = os.path.join('/files',
+                                 paths.HTTPD_NSS_CONF[1:],
+                                 'VirtualHost')
+        aug.set(
+            '{path}/#comment[. =~ regexp("NSSOCSP .*")]'.format(path=ocsp_path),
+            'NSSOCSP')
+        aug.rename('{path}/#comment[. = "NSSOCSP"]'.format(path=ocsp_path),
+                   'directive')
+        aug.set('{path}/directive[. = "NSSOCSP"]/arg'.format(path=ocsp_path),
+                'on')
```

Comment:
Nitpick: You can fit each of these statements on a single line by removing some unnecessary stuff:
```python
        aug.set('{}/#comment[.=~regexp("NSSOCSP .*")]'.format(path), 'NSSOCSP')
        aug.rename('{}/#comment[.="NSSOCSP"]'.format(path), 'directive')
        aug.set('{}/directive[.="NSSOCSP"]/arg'.format(path), 'on')
```